src/main/twirl/com/lightbend/lagom/docs/getstartedjavasbt.scala.html

```diff
@@ -19,81 +19,57 @@
 
 
 
-                    <p>
-                        If you have the prerequisites installed and are familiar with sbt, follow the steps below to create a Hello World project.
-                        Otherwise, see
-                        <a href="@context.path/documentation/@context.currentDocsVersion/java/JavaPrereqs.html">detailed
-                        instructions</a> for help setting up the prerequisites and getting started. Prerequisites for using Lagom with sbt include:
-
-                    <ul>
-                        <li>JDK 8</li>
-                        <li>sbt 0.13.13</li>
-                    </ul>
-                    </p>
-                    <p>The Lagom Giter8 template makes it easy to get started. The template downloads all dependencies and creates the basic project structure. It can take a few seconds to a few minutes to complete.</p>
-
-                      <p>After confirming the prerequisites, open a console or command window and follow these steps:</p>
-
-                  <ol>
-                    <li><p>Create a new directory for your project.</p></li>
-                        <li>
-                            <p>
-                                Change into the new directory and enter the following:
-                            </p>
+                    <p>Before starting, make sure that you
+                    have JDK 8 installed and configured. See <a
+                    href="@context.path/documentation/@context.currentDocsVersion/java/JavaPrereqs.html">Lagom documentation</a>
+                    for information on verifying your Java version.</p>
+                    <p>The Lightbend Tech Hub Project Starter makes it simple to create your first project. It uses a Lagom Giter8 template to generate a compressed file that contains: 
+                    </p><ul>
+                      <li>A distribution of sbt for building and running your project.</li> 
+                      <li>A Lagom Hello World project.</li>
+                    </ul> 
 
-                            <pre class="prettyprint"><code class="language-sh">sbt -Dsbt.version=0.13.15 new https://github.com/lagom/lagom-java.g8</code></pre>
-                        </li>
-                        <li>
-                            <p>
-                                The sbt Lagom template prompts for the following parameters. Press <code>Enter</code> to
-                                accept the defaults for the following:
+              <h2>Download the project from Tech Hub</h2>
+                    <p>Follow these steps to download a project created from the template:</p>
+                    <ol>
+                      <li><p>Open the <a class="" href="http://developer.lightbend.com/start/?group=lagom" target="_blank">Tech Hub Project Starter</a>.</p></li>
```

Comment:
Should it link directly to http://developer.lightbend.com/start/?group=lagom&project=lagom-java-sbt? This links to the index of all lagom templates currently. (Although, see also my [suggestion on #79](https://github.com/lagom/lagom.github.io/pull/79#issuecomment-315296791).)